packages/material-ui/src/useAutocomplete/useAutocomplete.test.js

```diff
@@ -1,164 +1,215 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
You can disable it in https://github.com/mui-org/material-ui/blob/f736ebe441d73c823919887ac0f8d5fa983ef0e0/.eslintrc.js#L192-L201 instead.